src/helpers/loginHelpers.js

```diff
@@ -0,0 +1,7 @@
+import * as requestHelpers from './requestHelpers';
+
+export const login = async (email, password) => {
+    const result = await requestHelpers.postData(`${process.env.REACT_APP_SERVER_URL}/api/login`, {email: email, password: password});
```

Comment:
could you please change this variable to just SERVER_URL, without REACT_APP part, because I did like this in heroku? :D

Reply:
That's how create-react-app setups things, you have to append REACT_APP to every env variable for it work out of the box. I could try to make it work with just SERVER_URL. Is it a difficult change on the heroku side?

Reply:
ouuu, I didnt know this :D okay, I will change it in heroku, its nor hard :)